### PR TITLE
Update configuration.mdx

### PR DESCRIPTION
**Pull request by schwichti**
More details on how to use `-backend-config`

For me, using -backend-config was not straight-forward. On the Internet you find other people who had the same misconceptions. Therefore, I think the documentation on this feature needs more clarification.

## Target Release

1.5.x

## Draft CHANGELOG entry

ENHANCEMENTS

### CODE REVIEW

**Comment by crw** [comment]
Thanks for this submission! I've notified the docs team, as it is a larger change they will need to do a review for style and correctness. Thanks again!

File: website/docs/language/settings/backends/configuration.mdx [hunk 1]
```diff
@@ -104,7 +104,32 @@ There are several ways to supply the remaining arguments:
...
+
+  ```hcl
+  # state.config
+  bucket = "your-bucket" 
+  key    = "your-state.tfstate"
+  region = "eu-central-1"
+  profile= "Your_Profile"
+  ```
+
+  Important: If your config file contains secrets it may be kept in
```
Comment (trujillo-adam) [inline website/docs/language/settings/backends/configuration.mdx:132]:
```suggestion
  When your configuration file contains secrets, you can store them in
```
Presumably, this is all important or else we wouldn't have written it down :)

**Review by trujillo-adam — COMMENTED** [review]
Thanks for adding this explanation. I tried to restate it in a way that's consistent with our writing styles. Let me know if I misunderstood.

**Review by crw — APPROVED** [review]

**Comment by crw** [comment]
I went ahead and committed the suggestions and will merge this PR. Thanks @trujillo-adam for the review.

**Pull request merged**